src/blocks/common.blocks/page/page.bemtree

```diff
@@ -24,6 +24,8 @@ block('page')(
 
         c.push({ elem: 'css' });
 
+        c.push({ elem: 'meta', attrs: { 'http-equiv': 'Content-Type', content: 'text/html; charset=utf-8' } });
```

Comment:
it's already there because of `page.bemhtml`: https://github.com/bem/bem-core/blob/v2/common.blocks/page/page.bemhtml#L14